spread.yaml

```diff
@@ -522,4 +522,44 @@ suites:
             distro_purge_package qemu genisoimage sshpass
             snap remove ubuntu-image
 
+    tests/performance/:
+        summary: Performance tests for snapd
+        manual: true
+        environment:
+            RES_FILE: "$PROJECT_PATH/test.perf"
+        prepare: |
+            . "$TESTSLIB/prepare.sh"
+            if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+                prepare_all_snap
+            else
+                prepare_classic
+            fi
+
+            . "$TESTSLIB/perf.sh"
+            create_results_file $RES_FILE
+
+        prepare-each: |
+            "$TESTSLIB/reset.sh" --reuse-core
+            . $TESTSLIB/prepare.sh
+            if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
+                prepare_each_classic
+            fi
+
+        restore: |
+            "$TESTSLIB/reset.sh" --store
+            if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
+                . $TESTSLIB/pkgdb.sh
+                distro_purge_package snapd
+                if [[ "$SPREAD_SYSTEM" != opensuse-* ]]; then
+                    # A snap-confine package never existed on openSUSE
+                    distro_purge_package snap-confine
+                fi
+            fi
+
+            . "$TESTSLIB/perf.sh"
```

Comment:
Can you highlight this as the distinctive feature so that the code above can be updated in sync with the rest.